NWEEI/NWEEI/Views/Programs/WaterConservation.cshtml

```diff
@@ -0,0 +1,184 @@
+                    </p>
+                </div>
+            </div>
+
+            <div class="content-highlight-container row">
+                <div class="content-highlight">
+                    <h6>Calculate Costs</h6>
+                    <p>
+                        Calculate water and cost savings and produce comprehensive
+                        cost/benefit analysis reports.
+                    </p>
+                </div>
+
+                <div class="content-highlight">
+                    <h6>Analyze Data</h6>
+                    <p>
+                        Monitor, collect, interpret and analyze data to evaluate
+                        the effectiveness of programs and modify them over time.
+                    </p>
+                </div>
+
+                <div class="content-highlight">
+                    <h6>Evaluate Usage Patterns</h6>
+                    <p>
+                        Evaluate water usage patterns for rural, urban, residential,
+                        and commercial sites; recommend efficiency measures as well
+                        as alternative water sources.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<!-- get real world experience -->
+<div class="container content-section d-flex">
+    <img src="~/images/water_adb_244643595.jpeg" class="content-image" />
+
+    <div class="content-info">
+        <h4>Get Real-World Experience</h4>
+        <p>
+            Employers are interested in more than just your degree. It’s important
+            that you’re able to show that you have relevant, real world work knowledge,
+            which is why many of your courses will include fieldwork. You’ll use the
+            knowledge you’ve gained in the classroom to help you succeed in hands-on
+            training opportunities at a partner organization in the Western United States.
+            We’ll work to match you with a utility or practitioner in your area,
+            after you’re enrolled in our program.
+        </p>
+    </div>
+</div>
+
+<!-- job opportunities -->
+<div class="light-block">
+    <div class="container">
+        <div class="content-section">
+            <h4>Job Opportunities</h4>
+            <p>
+                As a graduate in the Water Conservation Program, you’ll be ready to start
+                your career in water use and conservation. Our program continually seeks
+                out and participates in local, regional, and national networking opportunities
+                for one simple reason: to promote our students directly to those who have
+                the ability to provide jobs. Employment opportunities in Water Conservation
+                are excellent, especially if you are willing to relocate.
+            </p>
+            <p>
+                After you’ve earned your degree, you’ll be prepared to start working as:
+            </p>
+
```

Comment:
I feel like this might be another good spot to add the green buildings career map. We can ask Brenda, see what she thinks.